pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -332,9 +335,14 @@ func generateCloudConfig(configFile *ConfigFile) (cloudConfig *CloudConfig, err
 		return nil, err
 	}
 
+	// Determine if its a regional cluster
+	if configFile != nil && configFile.Global.Multimaster {
+		cloudConfig.Regional = true
```

Comment:
regional and multimaster have somewhat different meanings (I can imagine multi master but all in the same zone, for example). Can we converge on a single term and use it everywhere?

Reply:
Switched to using regional up front. Also added comment/warning at beginning over changes needed for HA/Zonal.